programs/programs/il/pe/member.py

```
@@ -302,3 +302,99 @@ def member_value(self, member):
+        member_dependency.SsdiReportedDependency,
+        # months_receiving_social_security_disability - not collected (see limitation #1)
+        # msp_countable_income (uses SSI methodology)
+        member_dependency.SsiEarnedIncomeDependency,
+        member_dependency.SsiUnearnedIncomeDependency,
+        # msp_asset_eligible
+        spm_dependency.CashAssetsDependency,
+        # state
+        household_dependency.IlStateCodeDependency,
+        # msp_standard_part_a_premium (for benefit value calculation)
+        member_dependency.MedicareQuartersOfCoverageDependency,
+    ]
```

Ruff (0.14.14)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)